R/exercise.R

```diff
@@ -33,9 +33,13 @@ setup_exercise_handler <- function(exercise_rx, session) {
   # setup reactive values for return
   rv <- reactiveValues(triggered = 0, result = NULL)
 
+  # debounce option to slow down successive exercise execution requests
+  debounce_s <- getOption("tutorial.exercise.debounce", 0) # value in seconds
```

Comment:
I think we can set a reasonably small default here, like 1 or 2 seconds.

```suggestion
  debounce_s <- getOption("tutorial.exercise.debounce", 1) # in seconds
```

Reply:
💙

Reply:
I'll update other changes to reflect it